examples/rabbitmq-service-basic-auth/rabbitmq_service_basic_auth.md

```diff
@@ -1,6 +1,6 @@
 # RabbitMQ service - Basic authentication
 
-The underlying connection of the consumer service is secured with basic authentication.
+RabbitMQ authentication deals with allowing a RabbitMQ client to connect to the server. After an application connects to RabbitMQ and before it can perform operations, it must authenticate, that is, present and prove its identity. In this example, the underlying connection of the listener is secured with Basic Authentication. A secured RabbitMQ listener is created by using the default host and port or custom configurations and providing the authentication details using the `rabbitmq:Credentials` record. Use it when you want to authenticate clients connections using username and password.
```

Comment:
```suggestion
RabbitMQ authentication deals with allowing a RabbitMQ client to connect to the server. After an application connects to RabbitMQ and before it can perform operations, it must authenticate (i.e., present and prove its identity). In this example, the underlying connection of the listener is secured with Basic Authentication. A secured RabbitMQ listener is created by using the default host and port or custom configurations and by providing the authentication details using the `rabbitmq:Credentials` record. Use it when you want to authenticate client connections using a username and password.
```